src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/GetAzureStorageAccountManagementPolicy.cs

```diff
@@ -0,0 +1,55 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Management.Storage.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.Azure.Management.Storage.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Management.Storage
+{
+    [Cmdlet(VerbsCommon.Get, StorageAccountManagementPolicyNounStr), OutputType(typeof(PSManagementPolicy))]
+    public class GetAzureStorageAccountManagementPolicyCommand : StorageAccountBaseCmdlet
+    {
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Resource Group Name.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Storage Account Name.")]
+        [Alias(AccountNameAlias)]
+        [ValidateNotNullOrEmpty]
+        public string StorageAccountName { get; set; }
```

Comment:
This should just be a "Name" you can add an alias to "StorageAccountName" if you want to.

Reply:
As the ManagementPolicy also has a property "Name", so don't want to ambiguous the Account name and Management policy Name , so use StorageAccountName  here.

Besides that, if use Name, pipe line the Policy object to remove-AzureRmStorageAccountManagementPolicy will fail, since the policy already has a property name, so the storage account name won't pipe to Name property.

Reply:
+1 to Pary's comment - piping the name by proeprty name is now deprecated, the defautl piping mechanism should be the InputObject parameter set (see comments below)

Reply:
Note that, for this cmdlet, it should also take piping by InputObject from Get-AzureRmStorageAccount